src/pyramid_jwt/policy.py

```diff
@@ -94,3 +93,15 @@ def forget(self, request):
             'has no effect.',
             stacklevel=3)
         return []
+
+    def _aud_string_to_list(self, audience):
+        """
+        Splits the audience variable into a
+        list to handle multiple audiences.
+        :param audience: Comma separated list of audiences
+        :return: List of one or more audiences
+        """
+        if audience is None:
+            return None
+        else:
+            return audience.split(',')
```

Comment:
Can you move that method out of the class into a separate function? Since it doesn't need to use anything from `self` there is no need for it to be a method.

I would also suggest splitting on something like `re.split(r'\s*,\s*`, audience)` to make this a bit friendlier.